glass/shapes.py

```diff
@@ -140,19 +140,19 @@ def ellipticity_ryden04(mu, sigma, gamma, sigma_gamma, size=None, *, rng=None):
 
     # default size if not given
     if size is None:
-        size = np.broadcast(gamma, sigma_gamma, mu, sigma).shape
+        size = np.broadcast(mu, sigma, gamma, sigma_gamma).shape
```

Comment:
Sorry what's going on here?

Reply:
I just did that to match the argument order in the function signature - 

https://github.com/glass-dev/glass/blob/59048a7d2fe6d981ab9232391b2792314241352a/glass/shapes.py#L100

nothing else 😬

Reply:
I see, okay. The order isn't important?

Reply:
Nope!